src/main/java/org/ays/common/model/AysPageable.java

```diff
@@ -35,10 +35,10 @@ public class AysPageable extends AysSort {
     private int page;
 
     /**
-     * Represents the page size.
-     * This value should be between 10 and 10, specifying the number of items per page.
+     * Represents the size of the page.
+     * The value must be between 1 and 10, specifying the number of items per page.
      */
-    @Range(min = 1, max = 99999999)
+    @Range(min = 1, max = 10)
```

Comment:
Sadece 10 olarak sabitlesek iyi olabilir gibi. Issue'da da bu şekilde yazılmıştı diye hatırlıyorum.
```suggestion
    @Range(min = 10, max = 10)
```

Reply:
Evet, pardon unutmuşum min değerini değiştirmeyi

Reply:
@agitrubard  testlerini hangi paketin içine yazmam gerekiyor ?

Reply:
> @agitrubard testlerini hangi paketin içine yazmam gerekiyor ?

@MenekseYuncu  Var olan listeleme servisleri için controller testlerini yazabiliriz. Bu şekilde validasyona takılırlar ve doğrulayabiliriz ✅